--- a/src/main-proxy.c
+++ b/src/main-proxy.c
@@ -461,12 +461,10 @@ static int http_aaa_setup(struct radius_
 
   char *user = 0;
   char *pwd = 0;
-#ifdef HAVE_OPENSSL
   char *ca = 0;
   char *cert = 0;
   char *key = 0;
   char *keypwd = 0;
-#endif
 
   req->radius = radius;
   
@@ -479,7 +477,6 @@ static int http_aaa_setup(struct radius_
     if (user && pwd) {
     }
     
-#ifdef HAVE_OPENSSL
     
     if (cert && strlen(cert)) {
 #if(_debug_)
@@ -530,7 +527,6 @@ static int http_aaa_setup(struct radius_
     
     curl_easy_setopt(curl, CURLOPT_SSL_VERIFYHOST, 0);
     curl_easy_setopt(curl, CURLOPT_SSLVERSION, CURL_SSLVERSION_DEFAULT);
-#endif
     
     curl_easy_setopt(curl, CURLOPT_VERBOSE, /*debug ? 1 :*/ 0);
     
